HdmiInput/HdmiInput.json

```diff
@@ -443,6 +500,32 @@
                     "frameRateD"
                 ]
             }
+        },
+        "hdmiGameFeatureStatusUpdate": {
+            "summary": "Triggered whenever game feature(ALLM) status changes for an HDMI Input",
```

Comment:
mentioned that depending on the game feature the response will change. so mode is not required always. In case of ALLM we return mode but for some other game status value could be something else.